translations/en-us.yml

```diff
@@ -1169,6 +1169,9 @@ preprints:
             description-input: 'Description'
         step-two:
             title: 'Metadata'
+            publication-doi-input: 'Publication Doi'
+            publication-year-input: 'Publication Year'
+            publication-citation-input: 'Publication Citation'
```

Comment:
If publication-citation is removed, let's remember to remove this translation string as well